### PR TITLE
Consul Associate 003 developer page

### PR DESCRIPTION
**Pull request by lszpunar**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-cert-update-consul-cert-page-hashicorp.vercel.app/certifications/networking-automation) 🔎

### CODE REVIEW

**Review by eddie-rowe — APPROVED** [review]
LGTM

Thank you for updating this!

File: src/content/certifications/programs/networking-automation.json [hunk 1]
```diff
@@ -13,9 +13,11 @@
 			"title": "Consul Associate",
 			"examCode": "003",
 			"productSlug": "consul",
-			"versionTested": "Consul 1.11 or higher",
+			"versionTested": "Product version tested: Consul 1.15 or higher",
 			"description": "The Consul Associate 003 is an upcoming certification for site reliability engineers (SREs), solutions architects (SAs), DevOps professionals, or other cloud engineers who know HashiCorp Consul fundamentals and have the skills to build, secure, and maintain Consul. Candidates will be best prepared for this exam if they have professional experience using Consul in production, but performing the exam objectives in a personal demo environment may also be sufficient. This person understands available Consul Enterprise features and can differentiate between Enterprise and Community Edition.",
 			"faqSlug": "consul-associate-003"
+			"links": {
+				"prepare": "/consul/tutorials/certification-003"
```
Comment (heatlikeheatwave) [inline src/content/certifications/programs/networking-automation.json:20]: Missing a closing bracket here. [The build](https://vercel.com/hashicorp/dev-portal/DRwuC5cRtBvaKWmSGqAwKGd5tQLg) should pass once this is fixed 
```suggestion
			"links": {
				"prepare": "/consul/tutorials/certification-003"
				}
```

File: src/content/certifications/exam-faqs/consul-associate-003.mdx [hunk 2]
```diff
@@ -2,19 +2,38 @@
...
+| 9   | Use gossip encryption |
+
+| ****NEW**** objectives in (003) |  |
+| ---- | -------------- |
+| 1c   | Summarize how Consul controls access to services at point of entry |
+| 1d   | Discuss how Consul automates networking tasks |
+| 2d   | Understand that Consul can run on multiple platforms |
+| 3c   | Configure and start Consul on Kubernetes |
+| 8    | Secure and connect service mesh applications at scale |
+| 9    | Monitor Consul |
```
Comment (lszpunar) [inline src/content/certifications/exam-faqs/consul-associate-003.mdx:35]:
```suggestion
| | (002) objectives ****NOT**** covered in (003) |
|  ---- | ------------ |
| 4    | Access the Consul key/value (KV) |

| | (002) objectives now covered within other objectives in (003) |
| ---- | --------------------- |
| 1   | Explain Consul Architecture |
| 2   | Deploy a single datacenter |
| 7   | Secure agent communication |
| 9   | Use gossip encryption |

| | ****NEW**** objectives in (003) |
| ---- | -------------- |
| 1c   | Summarize how Consul controls access to services at point of entry |
| 1d   | Discuss how Consul automates networking tasks |
| 2d   | Understand that Consul can run on multiple platforms |
| 3c   | Configure and start Consul on Kubernetes |
| 8    | Secure and connect service mesh applications at scale |
| 9    | Monitor Consul |
```

Comment (lszpunar) [reply src/content/certifications/exam-faqs/consul-associate-003.mdx:35]: image

**Review by AshNorman — CHANGES_REQUESTED** [review]
Amended requirements section

**Review by AshNorman — APPROVED** [review]
All good!

**Pull request merged**